TelCo.ColorCoder/EvenCountColorTest.cs

```diff
@@ -8,8 +8,16 @@ class EvenCountColortest
     {
         private static void Main(string[] args)
         {
-            ColorManual.printManual();
+            //Manual Print
+            ColorManual.PrintRefManual(new PrintManual());
 
+            //Behavioural Test for Reference Manual 
+            PrintManualTest testmanual = new PrintManualTest();
+            ColorManual.PrintRefManual(testmanual);
+            //Asserts that all 25 lines of Manual are printed
+            Debug.Assert(testmanual.WriteConsoleCall == 25);
```

Comment:
Next time, bring your test closer to the customer's need. Customers are not likely to be satisfied 'just' by having 25 rows on the manual.